source/component/tipue/tipuesearch.js.inc

```diff
@@ -366,7 +366,7 @@
 '                              ' + LineEnding +
 '                              if (set.imageZoom)' + LineEnding +
 '                              {' + LineEnding +
-'                                   out += ''<div id="tipue_search_image_modal"><div class="tipue_search_image_close">&#10005;</div><div class="tipue_search_image_block"><a id="tipue_search_zoom_url"><img id="tipue_search_zoom_img"></a><div id="tipue_search_zoom_text"></div></div></div>'';    ' + LineEnding +
+'                                   out += ''<div id="tipue_search_image_modal"><div class="tipue_search_image_close">&#10005;</div><div class="tipue_search_image_block">' + '<a id="tipue_search_zoom_url"><img id="tipue_search_zoom_img"></a><div id="tipue_search_zoom_text"></div></div></div>'';    ' + LineEnding +
```

Comment:
I have made a change in master branch of PasDoc that conflicts with this change now. Please resolve the conflict to leave this file unmodified on your branch.

( I would do it for you, but I do not have permission :) If you want to just allow me to do it, there should be a checkbox like "allow project maintainers to modify this PR" visible for you at submitting the PR. )

Reason:

I fixed `tipuesearch.js.inc` in https://github.com/pasdoc/pasdoc/commit/763370312ebd31f88187edfe85696eedb7808cde a bit differently. 

This file is generated from `tipuesearch.js` by `file_to_pascal_string` tool (also in this repo). And `file_to_pascal_string` since some time already contained logic to limit string literals to 255, exactly to work around this Delphi limit. The problem here was that `tipuesearch.js.inc` was made with older `file_to_pascal_string` version, so I just regenerated it by:

```
$ make build-tools # compiles file_to_pascal_string to bin
$ cd source/component/tipue/
$ make clean
$ make
$ make
../../../bin/file_to_pascal_data tipuesearch.css tipuesearch.css.inc
../../../bin/file_to_pascal_string tipuesearch.js tipuesearch.js.inc
../../../bin/file_to_pascal_data tipuesearch_set.js tipuesearch_set.js.inc
../../../bin/file_to_pascal_string tipue_results.html tipue_results.html.inc
../../../bin/file_to_pascal_data search.png search.png.inc
../../../bin/file_to_pascal_data loader.gif loader.gif.inc
../../../bin/file_to_pascal_data jquery.min.js jquery.min.js.inc
```

In effect it did the equivalent change to `tipuesearch.js.inc` but automatically :)

Reply:
Thanks for that! I wasn't aware it was generated. I've dropped my commit and rebased this branch onto master.